kratos/python_scripts/set_initial_state_process.py

```diff
@@ -72,9 +83,11 @@ def __init__(self, Model, settings):
             "mesh_id"         : 0,
             "model_part_name" : "please_specify_model_part_name",
             "dimension"       : 3,
-            "imposed_strain"  : ["0*t","0*t","0*t",0,0,0,0],
+            "imposed_strain_magnitude": "1.0",
+            "imposed_strain"  : ["0","0","0",0,0,0],
```

Comment:
Just to be sure, `imposed_strain_magnitude` is a multiplier for `imposed_strain` (same for the stres) right?. If this is the case, I'd name it `imposed_strain_multiplier` to make it clear.

Reply:
I was thinking in terms of direction and magnitude, but I agree with you. Done